docs/podman.1.md

```diff
@@ -94,8 +94,37 @@ has the capability to debug pods/images created by crio.
 | [podman-version(1)](podman-version.1.md)  | Display the Podman version information.                                        |
 | [podman-wait(1)](podman-wait.1.md)        | Wait on one or more containers to stop and print their exit codes.             |
 
+## FILES
+
+
+**libpod.conf** (`/etc/containers/libpod.conf`)
+
+	libpod.conf is the configuration file for all tools using libpod to manage containers
+
+**storage.conf** (`/etc/containers/storage.conf`)
+
+	storage.conf is the storage configuration file for all tools using containers/storage
+
+	The STORAGE configuration file specifies all of the available container storage options for tools using shared container storage, but in a TOML format that can be more easily modified and versioned.
+
+**mounts.conf** (`/usr/share/containers/mounts.conf` and optionally `/etc/containers/mounts.conf`)
```

Comment:
Does this make sense for the podman manpage? I don't think we use mounts.conf?

Reply:
mounts.conf is used in podman run. it mounts in the rhel subscription stuff so that you can do a 
yum update.

Reply:
I don't think it is? I don't see that code anywhere?

Reply:
Actually the patch seems to be missing which is far more troublesome.  @umohnani8 Could you get the secrets stuff from buildah merged into podman.

Reply:
yup working on it. Funny we did it for buildah but not podman.